### PR TITLE
Add QiDao/Mai BeethovenX LP pools on Fantom

### PR DESCRIPTION
**Pull request by 0xCmdrKeen**
Adds LP prices and APYs for two BeethovenX pools staked on QiDao:

- [MAI Concerto](https://beets.fi/#/pool/0x985976228a4685ac4ecb0cfdbeed72154659b6d900020000000000000000008d)
- [Qi Major](https://beets.fi/#/pool/0x7ae6a223cde3a17e0b95626ef71a2db5f03f540a00020000000000000000008a)

### CODE REVIEW

File: src/data/fantom/maiLpPools.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    "decimals": "1e18",
+    "tokens": [
+      {
+        "oracle": "tokens",
+        "oracleId": "USDC",
+        "decimals": "1e6"
+      },
+      {
+        "oracle": "tokens",
+        "oracleId": "miMATIC",
```
Comment (roman-monk) [inline src/data/fantom/maiLpPools.json:37]: i guess it should be MAI, we define it as MAI on fantom pools

File: src/api/stats/fantom/getMaiApys.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import pools from '../../../data/fantom/maiLpPools.json';
+import { beetClient } from '../../../apollo/client';
+import { fantomWeb3 } from '../../../utils/web3';
+
+const getMaiApys = async () =>
+  await getMasterChefApys({
+    web3: fantomWeb3,
+    chainId: 250,
+    masterchef: '0x230917f8a262bF9f2C3959eC495b11D1B7E1aFfC',
+    masterchefAbi: MasterChefAbi,
```
Comment (roman-monk) [inline src/api/stats/fantom/getMaiApys.js:12]: ABI is not needed (json file too)

**Comment by roman-monk** [comment]
Qidao is asking to not vault these

**Comment by 0xCmdrKeen** [comment]
> Qidao is asking to not vault these

I was going off of [@armads' message on Discord](https://discord.com/channels/755231190134554696/796840154991165451/970914754161934396) saying it actually was okay to vault these. 

Screen Shot 2022-05-16 at 10 04 15 AM

Has this changed?

**Pull request closed without merging**